src/drt/src/rp/FlexRP.h

```diff
@@ -124,6 +125,12 @@ class FlexRP
                                        frViaDef* viaDef2,
                                        bool isCurrDirX,
                                        frNonDefaultRule* ndr = nullptr);
+  void prep_via2viaForbiddenLen_minStepGF12(
```

Comment:
We should avoid putting more references to GF12 in the code. Is there some more generic name for this?

Reply:
This was already referent to GF12. This will be removed, since my implementation will probably override this one. But I will make sure this is true later, and do another PR with some refactorings.